### DISCUSSION TITLE
Why is process.exit(1) added in nodeHtmlToImage method?

### DISCUSSION THREAD

**Discussion by silhouette5366** [frinyvonnick/node-html-to-image#193] [Q&A] [answered]
hi everyone, 

I am using this library and sometimes I get timeout issues while converting HTML to images. Since this timeout issue rarely comes, I can manage that, but my main concern is that the server stops and POD restarts every time I get a timeout issue.

@frinyvonnick Any idea why you have added `process.exit(1)` in the **catch** statement at the end of the **nodeHtmlToImage** method? This is causing my server to stop and POD to restart.

Is there any purpose for adding `process.exit(1)`? What will happen if it is removed and the **nodeHtmlToImage** simply returns nothing in case of timeout issues?

**Comment by frinyvonnick** [comment, marked as answer]
Hello @silhouette5366 👋 When I created node-html-to-image it was for my own usage. My main usecase was creating thumbnails for blogposts on a gatsby app. It was called during the build process so in a CLI context, I wanted it to stop the process if something went wrong. I know some people have different usage for the library now. So feel free to open an issue/pull request about it 😄

**Reply by silhouette5366** [reply]
Thanks for the quick reply.
That was helpful.
Great work on the library by the way.
